src/message_validation.md

```diff
@@ -9,38 +9,49 @@
+| `ValSem5`      | Application messages must use ciphertext                    | ✅          | ✅     | `openmls/src/group/tests/test_validation.rs` |
+| `ValSem6`      | Ciphertext: decryption needs to work                        | ✅          | ✅     | `openmls/src/group/tests/test_validation.rs` |
+| `ValSem7`      | Membership tag presence                                     | ✅          | ✅     | `openmls/src/group/tests/test_validation.rs` |
+| `ValSem8`      | Membership tag verification                                 | ✅          | ✅     | `openmls/src/group/tests/test_validation.rs` |
+| `ValSem9`      | Confirmation tag presence                                   | ✅          | ✅     | `openmls/src/group/tests/test_validation.rs` |
+| `ValSem10`     | Signature verification                                      | ✅          | ✅     | `openmls/src/group/tests/test_validation.rs` |
+
+### Semantic validation of proposals covered by a Commit
+
+| ValidationStep | Description                                                                                 | Implemented    | Tested | Test File |
+| -------------- | ------------------------------------------------------------------------------------------- | -------------- | ------ | --------- |
+| `ValSem100`    | Add Proposal: Identity in proposals must be unique among proposals                          | ✅             | ❌     | 🈳        |
+| `ValSem101`    | Add Proposal: Signature public key in proposals must be unique among proposals              | ✅             | ❌     | 🈳        |
+| `ValSem102`    | Add Proposal: HPKE init key in proposals must be unique among proposals                     | ✅             | ❌     | 🈳        |
+| `ValSem103`    | Add Proposal: Identity in proposals must be unique among existing group members             | ✅             | ❌     | 🈳        |
+| `ValSem104`    | Add Proposal: Signature public key in proposals must be unique among existing group members | ✅             | ❌     | 🈳        |
+| `ValSem105`    | Add Proposal: HPKE init key in proposals must be unique among existing group members        | ✅             | ❌     | 🈳        |
+| `ValSem106`    | Add Proposal: required capabilities                                                         | ❌<sup>1</sup> | ❌     | 🈳        |
+| `ValSem107`    | Remove Proposal: Removed member must be unique among proposals                              | ✅             | ❌     | 🈳        |
+| `ValSem108`    | Remove Proposal: Removed member must be an existing group member                            | ✅             | ❌     | 🈳        |
+| `ValSem109`    | Update Proposal: Identity must be unchanged between existing member and new proposal        | ✅             | ❌     | 🈳        |
+| `ValSem110`    | Update Proposal: HPKE init key must be unique among existing members                        | ✅             | ❌     | 🈳        |
```

Comment:
Do you think the vacancy emoji 🈳 is too cryptic?

Reply:
Since it's technical documentation it should be as clear as possible to anyone. I think it's fine to not have anything in that column. Alternatively, you could use a common placeholder like "TBD" or "N/A".